packages/ui/src/components/organisms/ImageGallery/styles.scss

```diff
@@ -207,6 +207,19 @@
     }
   }
 
+  @mixin image_pos($pos) {
```

Comment:
It was a really clever way of resolving the duplication, but I find it hard to read. I would go with the common solution in this case :(